rust/src/client/mod.rs

```diff
@@ -6,15 +26,19 @@ use tokio::time;
 use crate::{
     crypto::ByteObject,
     mask::model::Model,
-    participant::{Participant, Task},
-    request::Proxy,
     sdk::api::CachedModel,
     service::Handle,
     CoordinatorPublicKey,
     InitError,
     PetError,
 };
 
+mod request;
+pub use request::Proxy;
+
+pub mod participant;
+pub use participant::{Participant, Task};
```

Comment:
to avoid duplication by public re-exports, the module should become private to the crate:
```suggestion
mod participant;
pub use participant::{Participant, Task};
```
this also avoids having a `Modules` and `Re-exports` section in the docs containing `participant.rs` and focuses the docs on this module docs.